framework/libra-framework/sources/block.move

```diff
@@ -229,11 +219,40 @@ module diem_framework::block {
         );
     }
 
+    fun maybe_advance_epoch(
+        vm: &signer,
+        timestamp: u64,
+        block_metadata_ref: &mut BlockResource,
+        round: u64
+        ) {
+            if (timestamp - reconfiguration::last_reconfiguration_time() >= block_metadata_ref.epoch_interval) {
```

Comment:
just for safety, let's check that the timestamp > last_reconfiguration_time (so that it's not zero or negative).